src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,40 @@
+package nextstep.qna.domain;
+
+import nextstep.users.domain.NsUser;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+public class Answers {
+    private final List<Answer> answers;
+
+    public Answers() {
+        this.answers = new ArrayList<>();
+    }
+
+    public Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public void add(Answer answer) {
+        answers.add(answer);
+    }
+
+    public boolean checkAllOwner(NsUser user) {
+        return Optional.ofNullable(answers)
+                .orElse(Collections.emptyList())
+                .stream()
+                .allMatch(answer -> answer.isOwner(user));
+    }
+
+    public List<DeleteHistory> deleteAll() {
+        List<DeleteHistory> deleteHistories = new ArrayList<>();
+        for (Answer answer : answers) {
+            deleteHistories.add(answer.delete());
+        }
+        return deleteHistories;
```

Comment:
Stream으로도 처리할 수 있겠네요 😄 
```suggestion
        return answers.map(...).toList();
```